pkgs/by-name/mi/miru/update.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gnused
+
+set -euo pipefail
+
+ROOT="$(dirname "$(readlink -f "$0")")"
+if [[ ! "$(basename $ROOT)" == "miru" || ! -f "$ROOT/package.nix" ]]; then
+    echo "ERROR: Not in the miru folder"
+    exit 1
+fi
+
+PACKAGE_NIX="$ROOT/package.nix"
+LINUX_NIX="$ROOT/linux.nix"
+DARWIN_NIX="$ROOT/darwin.nix"
+
+MIRU_LATEST_VER=$(curl -s "https://api.github.com/repos/ThaUnknown/miru/releases/latest" | jq -r .tag_name | sed 's/v//')
```

Comment:
Please check whether the request returned a valid result (e.g. if the repository's name changes the version tag will be `null`, raising a rather unclear error message from `nix-prefetch-url`, which could be prevented)

```
--- SHOWING ERROR LOG FOR miru-5.1.6 ----------------------

error: unable to download 'https://github.com/ThaUnknown/mirud/releases/download/vnull/linux-Miru-null.AppImage': HTTP error 404

       response body:

       Not Found
error: hash '' has wrong length for hash type 'sha256'


--- SHOWING ERROR LOG FOR miru-5.1.6 ----------------------
The update script for miru-5.1.6 failed with exit code 1
```